README.md

```diff
@@ -18,32 +18,98 @@ This oracle handles the following cases among others (let's say that our pair is
 - A/C and C/B are feeds (typically, WBTC/BTC and BTC/ETH).
 - A'/C and B/C are feeds, and there is an exchange rate between A and A'. (typically A=sDAI and A'=DAI).
 
-## Getting Started
+## WstETH/stETH Exchange Rate Adapter
 
-Install dependencies: `forge install`
+A specific implementation, the `WstEthStEthExchangeRateChainlinkAdapter`, provides the exchange rate between wstETH and stETH as a Chainlink-interface-compliant feed.
 
-In a `.env` file, set `ETH_RPC_URL` to the URL of an Ethereum provider.
+This adapter is deployed on Ethereum Mainnet at the address [0x905b7dAbCD3Ce6B792D874e303D336424Cdb1421](https://etherscan.io/address/0x905b7dabcd3ce6b792d874e303d336424cdb1421#code).
 
-Run test: `forge test`
+## MorphoChainlinkOracleV2Factory
```

Comment:
I would not do a separated category for the factory, and just add this to the corresponding oracle part